lib/ansible/module_utils/redfish_utils.py

```diff
@@ -511,6 +511,42 @@ def update_user_password(self, user):
             return response
         return {'ret': True}
 
+    def get_firmware_update_capabilities(self):
+        result = {}
+        response = self.get_request(self.root_uri + "/redfish/v1")
```

Comment:
Is it possible to not use `"/redfish/v1"` here but instead pass it from the module like we do with most other methods?

Reply:
Good catch on the `"/redfish/v1"`. I agree.